### PR TITLE
TC_02.03.021| Dashboard > Explanation text message

### PR DESCRIPTION
**Pull request by Stepwow**
https://trello.com/c/ET3P4nM0/345-tc02030201-dashboard-explanation-text-message

### CODE REVIEW

File: cypress/e2e/welcomeText.cy.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 ///<reference types ="cypress"/>
 import { welcomeMessage } from '../fixtures/welcomePageVerification.json';
+import { explanationText } from '../fixtures/welcomePageVerification.json';
 
 describe('welcomeText', () => {
     
     it('TC_02.03.019 Welcome to Jenkins massage verification', function () {
         cy.get('.empty-state-block h1').should('be.visible').and('have.text', welcomeMessage);
     }) 
+    it('TC_02.03.0201 Dashboard Explanation text message', function () {
```
Comment (S-Lanchik) [inline cypress/e2e/welcomeText.cy.js:10]: Check the number of your TC. I guess it should be TC_02.03.020 or 021.

Comment (Stepwow) [reply cypress/e2e/welcomeText.cy.js]: Could you please check code again. I made changes.

**Review by S-Lanchik — APPROVED** [review]

**Comment by S-Lanchik** [comment]
Just edit the PR name to the correct TC.

File: cypress/e2e/welcomeText.cy.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
 ///<reference types ="cypress"/>
 import { welcomeMessage } from '../fixtures/welcomePageVerification.json';
+import { explanationText } from '../fixtures/welcomePageVerification.json';
```
[the remark this answers is not in the document]
Comment (Stepwow) [reply cypress/e2e/welcomeText.cy.js:3]: Thank you!

**Pull request merged**